src/test/java/io/jenkins/plugins/jaxb/JAXBContextTest.java

```diff
@@ -31,8 +31,11 @@ private static void _smokes(JenkinsRule r) throws Throwable {
         Book book = new Book();
         book.setId(1L);
         book.setName("Guide to JAXB");
+        JAXBContext context;
+        try (SetContextClassLoader sccl = new SetContextClassLoader(RealJenkinsRule.Endpoint.class)) {
```

Comment:
If I understand what you are doing here,
```suggestion
        try (SetContextClassLoader sccl = new SetContextClassLoader(Book.class)) {
```
would be clearer.